internal/command/jsonplan/plan.go

```diff
@@ -682,6 +682,11 @@ func (p *Plan) marshalRelevantAttrs(plan *plans.Plan) error {
 
 		p.RelevantAttributes = append(p.RelevantAttributes, ResourceAttr{addr, path})
 	}
+
+	sort.Slice(p.RelevantAttributes, func(i, j int) bool {
+		return p.RelevantAttributes[i].Resource < p.RelevantAttributes[j].Resource
+	})
```

Comment:
I'd also add a small comment to explain why, because it's not trivial

Reply:
Fair point @eranelbaz, added some context in the comment.